### ISSUE TITLE
Inline generation not working

### ISSUE THREAD

**Issue by theshmo0** [Gregory-Jagermeister/Fantasy-Content-Generator#8]
The prompt gets triggered regardless of the callout symbol I choose, and I can also "hover select" an entry, but as I press enter/return to confirm my choice there is nothing pasted.

Is there any known issue regarding this?

**Comment by Christopher-Mazur** [comment]
Also running into this

**Comment by thedrowsygator** [comment]
I'm having a similar trouble with the inline use. It starts to work, then if you keep typing the option, the menu closes and you have to start over from the initial call-out.

**Comment by Gregory-Jagermeister** [comment]
Hi Sorry,

I've been really busy and could not get to the plugin for life reasons. Is there any particular time this occurs, my testing hasn't been able to replicate this (unless its on a canvas document, but that makes sense as this plugin was written just before Canvas came out haven't had a chance to update it to work with it yet).

Anymore information would be awesome as it could help me pin point the issue.

**Comment by thedrowsygator** [comment]
I hear that about life getting in the way. With so much happening everywhere, generating fantasy names feels like a low priority problem. I'll describe my situation as best I can, thank you for looking into this!

Stepping into the flow:

1. Open a note (existing or new), I need a name for an Orc, first and last.
2. Initiate the "Fantasy Content" via inline command "@"
3. Type "@O" dropdown appears with relevant options
4. To reduce scrolling, I continue typing "@Or" and the menu closes on the second letter typed
5. Two lay before me here:
  - If I continue typing the full phrase, go to a new line, then backspace once the menu will open with the option I want near/at the top
  - If I backspace over the second letter, the menu opens again with all "O" options (about 90%), but sometimes I need to restart the inline process with a fresh "@O" 

This might not be an issue with the plugin, just the way it works within the Obsidian platform.

It hasn't been a deal breaker, just a little bit of "oddness" in how I expected it to work, that's all. I'm treating the inline generation like a live list that will narrow down as I type.

Same example as above, expected results:

1. Open a note, I need a name for an Orc, first and last.
2. Start typing "@O" menu opens
3. Continue typing "@Or" menu refines to options that start with "Or"
4. Repeat step 3 until menu is narrowed to one (or none) options.
5. Select and insert or start over/backspace to get results

 
I think this describes it clearly, hit me up if I need to clarify anything. Again, thanks for your work and looking into this!

**Comment by Christopher-Mazur** [comment]
DrowsyGator's response sums it up well. 

I'll add one more caveat in case it happens to matter, I get this behavior on Mac's version of Obsidian. I haven't been able to test it on a PC to see if the behavior differs between platforms.

**Comment by thedrowsygator** [comment]
Good point, what I'm describing is happening in the Windows version of Obsidian.
